content/chainguard/chainguard-images/working-with-images/scanners/grype-tutorial/index.md

```diff
@@ -0,0 +1,250 @@
+
+{{< details "What is CVSS??" >}}
+{{< blurb/cvss >}}
+{{< /details >}}
+
+In our output, we can see that we have 0 critical, 1 high, and 8 medium CVEs:
+
+```
+   ├── by severity: 0 critical, 1 high, 8 medium, 0 low, 0 negligible 
+```
+grype also counts the number of CVE by fixed status. If a CVE is marked as fixed, it can be resolved by updating to a newer version of the package. Our output suggests that 7 packages have been fixed and can be remediated with updates:
+
+```
+   └── by status:   7 fixed, 3 not-fixed, 0 ignored 
+```
+### Itemized CVEs
+
+In addition to the summary, grype provides an itemized list of CVE. By default, these are in table format, and list the pakage name, current version, the package type (such as apt, apk, binary), and severity. If the package is fixed, grype will also indicate the package version where the fix was introduced.
+
+grype writes itemized CVE to stdout, so the report of itemized CVE can easily be redirected to a file:
+
+```bash
+grype python:3.10.14-alpine3.20 > report.txt
+```
+Alternately, you can use the `--file` flag to write to a file:
+
+
+```bash
+grype --file report.txt python:3.10.14-alpine3.20
+```
+
+Redirecting output can also be useful to suppress a long list of CVE so that the summary can more easily be viewed.
+
+## Output Formats
+
+### Standard Formats
+
+Itemized CVEs can be written to a number of formats, including the XML- or JSON-based [cyclonedx](https://cyclonedx.org/) SBOM standard and the [SARIF](https://sarifweb.azurewebsites.net/) static analysis format. To maximize the information provided by grype, it is recommended to use the JSON output type:
+
+```bash
+ grype -o json python:3.10.14-alpine3.20 > report.json
+```
+
+When using these more detailed formats, grype provides additional useful fields, such as the data source of the CVE, URLs to information on the CVE, advisories, related vulnerabilities, and details on how the vulnerability was detected.
+
+### Output Templates
+
+Additional output formats are available as Hugo templates. These include output templates for HTML and CSV, and a [full list](https://github.com/anchore/grype/tree/main/templates) can be found at the grype GitHub repository.
+
+To generate a HTML file of the itemized CVEs, first clone the grype repository from GitHub, then provide the path to the template file in your grype command:
+
+```bash
+git clone git@github.com:anchore/grype.git ~/.grype
+grype -o template -t ~/.grype/templates/html.tmpl python:3.10.14-alpine3.20 > report.html
+```
+![Screenshot of rendered HTML generated by the included grype HTML template](grype_html_output.png)
+
+To generate a CSV:
+
+```bash
+git clone git@github.com:anchore/grype.git ~/.grype
+grype -o template -t ~/.grype/templates/csv.tmpl python:3.10.14-alpine3.20 > report.csv
+```
+
+## grype Explain
```

Comment:
```suggestion
## `grype explain`
```

this is a weird case, but I think rendering it as code makes sense here

Reply:
Yes, looks slightly odd, but it's more semantic.